frontend/packages/dev-console/src/components/topology/knative-overview.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ListGroup } from 'patternfly-react';
+
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import {
+  ResourceLink,
+  SidebarSectionHeading,
+  ExternalLink,
+} from '@console/internal/components/utils';
+import { OverviewItem } from '@console/internal/components/overview';
+import { ConfigurationModel, KsRouteModel, RevisionModel } from '../../models';
+
+const OverviewDetailsKnativeResourcesTab: React.SFC<OverviewDetailsResourcesTabProps> = ({
+  item: { ksroutes, revisions, configurations },
+}) => (
+  <div className="overview__sidebar-pane-body">
+    <KnativeOverview ksroutes={ksroutes} configurations={configurations} revisions={revisions} />
+  </div>
+);
+
+const RevisionsOverviewListItem: React.SFC<RevisionsOverviewListItemProps> = ({ revision }) => {
```

Comment:
You can simplify the return value by deconstructing in the function params:
```suggestion
const RevisionsOverviewListItem: React.SFC<RevisionsOverviewListItemProps> = ({ revision: { metadata: { name, namespace }}) => {
```